src/components/LenderProfile/LenderBadges.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+	<div>
+		<h4>{{ lenderName }}'s Badges</h4>
```

Comment:
We are adding a `no-name` value like here: https://github.com/kiva/ui/blob/0d5bf7ba307a528a7c94956dc3a24869867d104e/src/components/LenderProfile/LenderTeamsList.vue#L89